WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -127,4 +127,13 @@ def users_to_sentence(users, include_email: false)
       include_email ? mail_to(user.email, user.name) : user.name
     end.xss_aware_to_sentence
   end
+
+  def region_option_tags(selected_id:)
```

Comment:
It's a keyword argument, but now I guess it should be `nil` by default, otherwise it will raise an exception when we don't pass it and use below.